src/main/test/java/org/skypro/list/service/DepartmentServiceTest.java

```diff
@@ -0,0 +1,75 @@
+package org.skypro.list.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.skypro.list.employee.Employee;
+
+import java.util.*;
+import java.util.stream.DoubleStream;
+
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class DepartmentServiceTest {
+
+    @Mock
+    private EmployeeServiceImpl employeeService;
+    @InjectMocks
+    private DepartmentServiceImpl departmentService;
+    private List<Employee> employees;
+
+@Test
+void DepartmentServiceIml(){}
+
+    @BeforeEach
+    void init() {
+      List<Employee> employees = Arrays.asList(new Employee("Ivan", "Petrov", 1, 22_500),
```

Comment:
Это у тебя локальная переменная получилась, а в моки передаешь null
```suggestion
      employees = Arrays.asList(new Employee("Ivan", "Petrov", 1, 22_500),
```